docs/source/markdown/options/domainname.md

```diff
@@ -0,0 +1,9 @@
+####> This option file is used in:
+####>   podman create, run
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--domainname**=*name*
+
+Container domain name
+
+Sets the container domain name that is available inside the container. Can only be used with a private UTS namespace `--uts=private` (default). This option is only curently only available in rootful mode.
```

Comment:
typo in `curently`

Why is it only available for root?  We run the tests we have added for crun also with unprivileged users

Reply:
leftover from the first implementation, ill change